libs/components/forms/src/lib/modules/shared/sky-forms-resources.module.ts

```diff
@@ -92,6 +92,15 @@ const RESOURCES: { [locale: string]: SkyLibResources } = {
     skyux_file_attachment_file_upload_link_removed: {
       message: 'Link to {0} removed.',
     },
+    skyux_file_attachment_file_type_error_label_text: {
+      message: 'Please upload a file of type {0}',
+    },
+    skyux_file_attachment_max_file_size_error_label_text: {
+      message: 'Please upload a file under {0}Kbs',
```

Comment:
Change "Kbs" to "KB" (all caps) which is the abbreviation for 1024 bytes. There's no need for the "s" at the end.